pkg/controller/v1beta1/inferenceservice/components/predictor.go

```diff
@@ -63,6 +63,8 @@ func NewPredictor(client client.Client, scheme *runtime.Scheme, inferenceService
 func (p *Predictor) Reconcile(isvc *v1beta1.InferenceService) (ctrl.Result, error) {
 	var container *v1.Container
 	var podSpec v1.PodSpec
+	sRuntimeLabels := map[string]string{}
+	sRuntimeAnnotations := map[string]string{}
```

Comment:
```suggestion
	var sRuntimeLabels map[string]string
	var sRuntimeAnnotations map[string]string
```
No need to create empty maps here